features/facebookImport/src/components/advertisingValueMiniStory/advertisingValueMiniStory.jsx

```diff
@@ -1,45 +1,12 @@
-import React, { useRef } from "react";
+import React from "react";
 import i18n from "../../i18n";
 import ListOfDetails from "../listOfDetails/listOfDetails.jsx";
 import "./advertisingValueMiniStory.css";
 
-const calculateFontSize = (text, maxWidth) => {
```

Comment:
What's happened here?

Reply:
Wasn't used actually and is a weird way to do stuff, which now with the changed card also doesn't work anymore. So I got rid of it while making sure that nothing really changes